.github/workflows/weasel.yml

```diff
@@ -0,0 +1,35 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: Weasel License checks
+
+on:
+  push:
+  create:
+  pull_request:
+    types: [opened, reopened, edited, synchronize]
+
+jobs:
+  weasel:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+    - name: Run weasel
+      uses: comcast/weasel@v0.3
```

Comment:
It looks like #4760 (weasel v0.4) [breaks .dependency_license](https://github.com/apache/trafficcontrol/commit/931a620615#diff-4bcf5e086632706ea5144a1c6e27244bL33-R133) for weasel versions 0.3 and older, because [comcast/weasel@v0.3](/comcast/weasel/tree/v0.3) does not recognize `Apache-2.0` as a valid license. Because of that, the GitHub action lists errors on a bunch of files from `.dependency_license`.

<details><summary>GitHub Actions log for <code>comcast/weasel@v0.3</code> (click to expand)</summary>

```
2020-06-30T02:20:38.2912804Z [command]/usr/bin/git log -1
2020-06-30T02:20:38.2962402Z commit a06ecf3e27f34345ae1dfd7e384d3d4b83f6907b
2020-06-30T02:20:38.2963003Z Author: ocket8888 <ocket8888@apache.org>
2020-06-30T02:20:38.2963868Z Date:   Mon Jun 29 20:09:27 2020 -0600
2020-06-30T02:20:38.2964304Z 
2020-06-30T02:20:38.2964717Z     Added badge indicator to README
2020-06-30T02:20:38.3066463Z ##[group]Run comcast/weasel@v0.3
2020-06-30T02:20:38.3066640Z ##[endgroup]
2020-06-30T02:20:38.3094199Z ##[command]/usr/bin/docker run --name d3ea4b73f872b643369204b6c9aded8b72_610865 --label 3888d3 --workdir /github/workspace --rm -e HOME -e GITHUB_JOB -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_REPOSITORY_OWNER -e GITHUB_RUN_ID -e GITHUB_RUN_NUMBER -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_SERVER_URL -e GITHUB_API_URL -e GITHUB_GRAPHQL_URL -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e ACTIONS_CACHE_URL -e GITHUB_ACTIONS=true -e CI=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/atc-gha/atc-gha":"/github/workspace" 3888d3:ea4b73f872b643369204b6c9aded8b72
2020-06-30T02:21:24.2852530Z Error                              Apache-2.0! blueprints/img/multi-interface-ui-sample.png
2020-06-30T02:21:24.2853031Z Error                              Apache-2.0! blueprints/img/profile-param-compare-manage.png
2020-06-30T02:21:24.2853426Z Error                              Apache-2.0! docs/source/admin/images/bad.png
2020-06-30T02:21:24.2867920Z Error                              Apache-2.0! docs/source/admin/images/good.png
2020-06-30T02:21:24.2868884Z Error                              Apache-2.0! docs/source/admin/quick_howto/anonymous_blocking/01.png
2020-06-30T02:21:24.2869706Z Error                              Apache-2.0! docs/source/admin/quick_howto/anonymous_blocking/02.png
2020-06-30T02:21:24.2870393Z Error                              Apache-2.0! docs/source/admin/quick_howto/anonymous_blocking/03.png
2020-06-30T02:21:24.2870985Z Error                              Apache-2.0! docs/source/admin/quick_howto/cachegroup_fallback/00.png
2020-06-30T02:21:24.2871603Z Error                              Apache-2.0! docs/source/admin/quick_howto/cachegroup_fallback/01.png
2020-06-30T02:21:24.2872220Z Error                              Apache-2.0! docs/source/admin/quick_howto/cachegroup_fallback/02.png
2020-06-30T02:21:24.2872839Z Error                              Apache-2.0! docs/source/admin/quick_howto/cachegroup_fallback/03.png
2020-06-30T02:21:24.2873448Z Error                              Apache-2.0! docs/source/admin/quick_howto/cachegroup_fallback/04.png
2020-06-30T02:21:24.2874051Z Error                              Apache-2.0! docs/source/admin/quick_howto/cachegroup_fallback/05.png
2020-06-30T02:21:24.2875037Z Error                              Apache-2.0! docs/source/admin/quick_howto/content_invalidation/01.png
2020-06-30T02:21:24.2875668Z Error                              Apache-2.0! docs/source/admin/quick_howto/content_invalidation/02.png
2020-06-30T02:21:24.2876286Z Error                              Apache-2.0! docs/source/admin/quick_howto/content_invalidation/03.png
2020-06-30T02:21:24.2876875Z Error                              Apache-2.0! docs/source/admin/quick_howto/dnssec/00.png
2020-06-30T02:21:24.2877469Z Error                              Apache-2.0! docs/source/admin/quick_howto/dnssec/01.png
2020-06-30T02:21:24.2878061Z Error                              Apache-2.0! docs/source/admin/quick_howto/dnssec/02.png
2020-06-30T02:21:24.2878647Z Error                              Apache-2.0! docs/source/admin/quick_howto/dnssec/03.png
2020-06-30T02:21:24.2879238Z Error                              Apache-2.0! docs/source/admin/quick_howto/dnssec/04.png
2020-06-30T02:21:24.2879824Z Error                              Apache-2.0! docs/source/admin/quick_howto/dnssec/05.png
2020-06-30T02:21:24.2882451Z Error                              Apache-2.0! docs/source/admin/quick_howto/dnssec/06.png
2020-06-30T02:21:24.2882863Z Error                              Apache-2.0! docs/source/admin/quick_howto/federations/01.png
2020-06-30T02:21:24.2883469Z Error                              Apache-2.0! docs/source/admin/quick_howto/federations/02.png
2020-06-30T02:21:24.2883932Z Error                              Apache-2.0! docs/source/admin/quick_howto/federations/03.png
2020-06-30T02:21:24.2884323Z Error                              Apache-2.0! docs/source/admin/quick_howto/federations/04.png
2020-06-30T02:21:24.2884709Z Error                              Apache-2.0! docs/source/admin/quick_howto/multi_site/00.png
2020-06-30T02:21:24.2885097Z Error                              Apache-2.0! docs/source/admin/quick_howto/multi_site/01.png
2020-06-30T02:21:24.2885480Z Error                              Apache-2.0! docs/source/admin/quick_howto/multi_site/02.png
2020-06-30T02:21:24.2885875Z Error                              Apache-2.0! docs/source/admin/quick_howto/multi_site/03.png
2020-06-30T02:21:24.2886269Z Error                              Apache-2.0! docs/source/admin/quick_howto/multi_site/ds_profile.png
2020-06-30T02:21:24.2886672Z Error                              Apache-2.0! docs/source/admin/quick_howto/multi_site/ds_profile_parameters.png
2020-06-30T02:21:24.2887075Z Error                              Apache-2.0! docs/source/admin/quick_howto/multi_site/mso-enable.png
2020-06-30T02:21:24.2887507Z Error                              Apache-2.0! docs/source/admin/quick_howto/profile_compare_mgmt/compare_profiles_menu.png
2020-06-30T02:21:24.2887947Z Error                              Apache-2.0! docs/source/admin/quick_howto/profile_compare_mgmt/compare_profiles_table.png
2020-06-30T02:21:24.2888385Z Error                              Apache-2.0! docs/source/admin/quick_howto/profile_compare_mgmt/select_profiles_dialog.png
2020-06-30T02:21:24.2888784Z Error                              Apache-2.0! docs/source/admin/quick_howto/regionalgeo/01.png
2020-06-30T02:21:24.2889175Z Error                              Apache-2.0! docs/source/admin/quick_howto/regionalgeo/02.png
2020-06-30T02:21:24.2889566Z Error                              Apache-2.0! docs/source/admin/quick_howto/regionalgeo/03.png
2020-06-30T02:21:24.2889986Z Error                              Apache-2.0! docs/source/admin/quick_howto/server_capability/ds_server_caps_table.png
2020-06-30T02:21:24.2890406Z Error                              Apache-2.0! docs/source/admin/quick_howto/server_capability/server_caps_table.png
2020-06-30T02:21:24.2890834Z Error                              Apache-2.0! docs/source/admin/quick_howto/server_capability/server_server_caps_table.png
2020-06-30T02:21:24.2891224Z Error                              Apache-2.0! docs/source/admin/quick_howto/static_dns/00.png
2020-06-30T02:21:24.2891609Z Error                              Apache-2.0! docs/source/admin/quick_howto/static_dns/01.png
2020-06-30T02:21:24.2892145Z Error                              Apache-2.0! docs/source/admin/quick_howto/steering/01.png
2020-06-30T02:21:24.2892517Z Error                              Apache-2.0! docs/source/admin/quick_howto/steering/02.png
2020-06-30T02:21:24.2892907Z Error                              Apache-2.0! docs/source/admin/quick_howto/steering/03.png
2020-06-30T02:21:24.2893317Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/changelog_dialog.png
2020-06-30T02:21:24.2893720Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/changelog_page.png
2020-06-30T02:21:24.2894125Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/logout_button.png
2020-06-30T02:21:24.2894531Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/popout_button.png
2020-06-30T02:21:24.2894933Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/profile_button.png
2020-06-30T02:21:24.2895354Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/release_info_button.png
2020-06-30T02:21:24.2896237Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu.png
2020-06-30T02:21:24.2896705Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu_configure.png
2020-06-30T02:21:24.2897112Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu_monitor.png
2020-06-30T02:21:24.2897496Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu_other.png
2020-06-30T02:21:24.2897901Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu_services.png
2020-06-30T02:21:24.2898308Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu_tools.png
2020-06-30T02:21:24.2898720Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu_topology.png
2020-06-30T02:21:24.2899129Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_menu_user_admin.png
2020-06-30T02:21:24.2899540Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_table_ds.png
2020-06-30T02:21:24.2899955Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_table_ds_requests.png
2020-06-30T02:21:24.2900359Z Error                              Apache-2.0! docs/source/admin/traffic_portal/images/tp_table_server.png
2020-06-30T02:21:24.2900759Z Error                              Apache-2.0! docs/source/admin/traffic_router/images/edge_tr_example.png
2020-06-30T02:21:24.2901123Z Error                              Apache-2.0! docs/source/basics/fwda.png
2020-06-30T02:21:24.2901530Z Error                              Apache-2.0! docs/source/development/debugging/to_perl_connection_breakpoint.png
2020-06-30T02:21:24.2901945Z Error                              Apache-2.0! docs/source/development/traffic_monitor/Health_Pipeline.png
2020-06-30T02:21:24.2902331Z Error                              Apache-2.0! docs/source/development/traffic_monitor/Monitor_Pipeline.png
2020-06-30T02:21:24.2902748Z Error                              Apache-2.0! docs/source/development/traffic_monitor/Ops-Config_Pipeline.png
2020-06-30T02:21:24.2903154Z Error                              Apache-2.0! docs/source/development/traffic_monitor/Peer_Pipeline.png
2020-06-30T02:21:24.2903550Z Error                              Apache-2.0! docs/source/development/traffic_monitor/Pipeline.png
2020-06-30T02:21:24.2903951Z Error                              Apache-2.0! docs/source/development/traffic_monitor/Stat_Pipeline.png
2020-06-30T02:21:24.2904313Z Error                              Apache-2.0! docs/source/docutils.conf
2020-06-30T02:21:24.2904665Z Error                              Apache-2.0! docs/source/favicon.ico
2020-06-30T02:21:24.2905196Z Error                              Apache-2.0! docs/source/overview/images/cg_hierarchy.png
2020-06-30T02:21:24.2905583Z Error                              Apache-2.0! docs/source/overview/images/topography.png
2020-06-30T02:21:24.2905996Z Error                              Apache-2.0! docs/source/overview/images/traffic_control_overview_3.png
2020-06-30T02:21:24.2906360Z Error                              Apache-2.0! docs/wiki/GitHub-Flow.png
2020-06-30T02:21:24.2906732Z Error                              Apache-2.0! experimental/traffic-portal-angular8/angular.json
2020-06-30T02:21:24.2907129Z Error                              Apache-2.0! experimental/traffic-portal-angular8/e2e/tsconfig.json
2020-06-30T02:21:24.2907524Z Error                              Apache-2.0! experimental/traffic-portal-angular8/package-lock.json
2020-06-30T02:21:24.2907916Z Error                              Apache-2.0! experimental/traffic-portal-angular8/package.json
2020-06-30T02:21:24.2908334Z Error                              Apache-2.0! experimental/traffic-portal-angular8/src/assets/manifest.webmanifest
2020-06-30T02:21:24.2908736Z Error                              Apache-2.0! experimental/traffic-portal-angular8/src/tsconfig.app.json
2020-06-30T02:21:24.2909223Z Error                              Apache-2.0! experimental/traffic-portal-angular8/src/tsconfig.server.json
2020-06-30T02:21:24.2909670Z Error                              Apache-2.0! experimental/traffic-portal-angular8/src/tsconfig.spec.json
2020-06-30T02:21:24.2910065Z Error                              Apache-2.0! experimental/traffic-portal-angular8/src/tslint.json
2020-06-30T02:21:24.2910458Z Error                              Apache-2.0! experimental/traffic-portal-angular8/tsconfig.app.json
2020-06-30T02:21:24.2910849Z Error                              Apache-2.0! experimental/traffic-portal-angular8/tsconfig.json
2020-06-30T02:21:24.2911226Z Error                              Apache-2.0! experimental/traffic-portal-angular8/tsconfig.spec.json
2020-06-30T02:21:24.2911622Z Error                              Apache-2.0! experimental/traffic_ops_auth/config.json
2020-06-30T02:21:24.2912005Z Error                              Apache-2.0! experimental/traffic_router_golang/cfg.json
2020-06-30T02:21:24.2912383Z Error                              Apache-2.0! experimental/webfront/rules.json
2020-06-30T02:21:24.2912752Z Error                              Apache-2.0! experimental/webfront/server.pem
2020-06-30T02:21:24.2913136Z Error                              Apache-2.0! experimental/webfront/simpleserver/server.pem
2020-06-30T02:21:24.2913513Z Error                              Apache-2.0! experimental/webfront/webfront.config
2020-06-30T02:21:24.2913865Z Error                              Apache-2.0! grove/conf/grove.cfg
2020-06-30T02:21:24.2914207Z Error                              Apache-2.0! grove/grove.cfg
2020-06-30T02:21:24.2914584Z Error                              Apache-2.0! grove/grovetccfg/grove_profile.traffic_ops
2020-06-30T02:21:24.2914937Z Error                              Apache-2.0! grove/integration_test/grove.cfg
2020-06-30T02:21:24.2915320Z Error                              Apache-2.0! grove/integration_test/remap-base-test.json
2020-06-30T02:21:24.2915728Z Error                              Apache-2.0! grove/integration_test/tests/plugins/modify_headers/remap.json
2020-06-30T02:21:24.2916145Z Error                              Apache-2.0! grove/integration_test/tests/plugins/range_req_handler/remap.json
2020-06-30T02:21:24.2916510Z Error                              Apache-2.0! grove/remap.json
2020-06-30T02:21:24.2916881Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/enroller/server_template.json
2020-06-30T02:21:24.2917292Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_monitor/traffic_monitor.cfg
2020-06-30T02:21:24.2917732Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cachegroups/010-TRAFFIC_ANALYTICS.json
2020-06-30T02:21:24.2918296Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cachegroups/020-TRAFFIC_OPS.json
2020-06-30T02:21:24.2918741Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cachegroups/030-TRAFFIC_OPS_DB.json
2020-06-30T02:21:24.2920188Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cachegroups/040-TRAFFIC_PORTAL.json
2020-06-30T02:21:24.2920652Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cachegroups/050-TRAFFIC_STATS.json
2020-06-30T02:21:24.2921076Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cachegroups/060-ciabMid.json
2020-06-30T02:21:24.2921502Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cachegroups/070-ciabEdge.json
2020-06-30T02:21:24.2921922Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/cdns/010-CDN-in-a-Box.json
2020-06-30T02:21:24.2922379Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/deliveryservice_servers/010-demo1.json
2020-06-30T02:21:24.2923210Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/deliveryservices/010-ciab.json
2020-06-30T02:21:24.2923723Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/divisions/010-north.json
2020-06-30T02:21:24.2924149Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/divisions/020-usa.json
2020-06-30T02:21:24.2924557Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/phys_locations/010-apachecon.json
2020-06-30T02:21:24.2924994Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/phys_locations/020-phys_location.json
2020-06-30T02:21:24.2925421Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/000-GLOBAL.json
2020-06-30T02:21:24.2925851Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/010-ATS_EDGE_TIER_CACHE.json
2020-06-30T02:21:24.2926292Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/020-ATS_MID_TIER_CACHE.json
2020-06-30T02:21:24.2926720Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/030-BIND_ALL.json
2020-06-30T02:21:24.2927139Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/040-CCR_CIAB.json
2020-06-30T02:21:24.2927559Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/050-ENROLLER_ALL.json
2020-06-30T02:21:24.2927994Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/070-RASCAL-Traffic_Monitor.json
2020-06-30T02:21:24.2928540Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/080-TRAFFIC_STATS.json
2020-06-30T02:21:24.2928971Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/081-INFLUXDB.json
2020-06-30T02:21:24.2929397Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/profiles/082-GRAFANA.json
2020-06-30T02:21:24.2929811Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/regions/010-dc.json
2020-06-30T02:21:24.2930228Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/regions/020-montreal.json
2020-06-30T02:21:24.2930701Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/servers/010-dns_server.json
2020-06-30T02:21:24.2931118Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/servers/020-db_server.json
2020-06-30T02:21:24.2931690Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/servers/030-enroller_server.json
2020-06-30T02:21:24.2932115Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/tenants/010-tenant01.json
2020-06-30T02:21:24.2932542Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/tenants/020-tenant02.json
2020-06-30T02:21:24.2932966Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/tenants/030-tenant11.json
2020-06-30T02:21:24.2933388Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/tenants/040-tenant12.json
2020-06-30T02:21:24.2933789Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/tenants/050-tenant21.json
2020-06-30T02:21:24.2934201Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/tenants/060-tenant22.json
2020-06-30T02:21:24.2934618Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/types/010-BIND.json
2020-06-30T02:21:24.2935038Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/types/020-ENROLLER.json
2020-06-30T02:21:24.2938492Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/types/050-GRAFANA.json
2020-06-30T02:21:24.2939096Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/users/010-tmonitor.json
2020-06-30T02:21:24.2939515Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/users/020-tvault.json
2020-06-30T02:21:24.2939926Z Error                              Apache-2.0! infrastructure/cdn-in-a-box/traffic_ops_data/users/030-tstats.json
2020-06-30T02:21:24.2940331Z Error                              Apache-2.0! infrastructure/docker/traffic_ops/profile.origin.traffic_ops
2020-06-30T02:21:24.2940799Z Error                              Apache-2.0! misc/logos/ATC-PNG-FULL-BLACK-LARGE.png
2020-06-30T02:21:24.2941238Z Error                              Apache-2.0! misc/logos/ATC-PNG-FULL-BLACK.png
2020-06-30T02:21:24.2941621Z Error                              Apache-2.0! misc/logos/ATC-PNG-FULL-WHITE-LARGE.png
2020-06-30T02:21:24.2941974Z Error                              Apache-2.0! misc/logos/ATC-PNG-FULL-WHITE.png
2020-06-30T02:21:24.2942340Z Error                              Apache-2.0! misc/logos/ATC-PNG-LARGE.png
2020-06-30T02:21:24.2942699Z Error                              Apache-2.0! misc/logos/ATC-PNG.png
2020-06-30T02:21:24.2943062Z Error                              Apache-2.0! misc/logos/ATC-SVG-FULL.sketch
2020-06-30T02:21:24.2943421Z Error                              Apache-2.0! misc/logos/ATC-SVG.sketch
2020-06-30T02:21:24.2943807Z Error                              Apache-2.0! misc/logos/page_logos/trafficLogoFactory.psd
2020-06-30T02:21:24.2944199Z Error                              Apache-2.0! misc/logos/page_logos/traffic_monitor_logo.png
2020-06-30T02:21:24.2944588Z Error                              Apache-2.0! misc/logos/page_logos/traffic_operations_logo.png
2020-06-30T02:21:24.2944977Z Error                              Apache-2.0! misc/logos/page_logos/traffic_portal.png
2020-06-30T02:21:24.2945364Z Error                              Apache-2.0! misc/logos/page_logos/traffic_router_logo.png
2020-06-30T02:21:24.2945728Z Error                              Apache-2.0! misc/logos/page_logos/traffic_stats_logo.png
2020-06-30T02:21:24.2946116Z Error                              Apache-2.0! misc/logos/page_logos/traffic_stats_logo_red.png
2020-06-30T02:21:24.2946493Z Error                              Apache-2.0! misc/traffic-control-cdn/favicon.ico
2020-06-30T02:21:24.2946873Z Error                              Apache-2.0! misc/traffic-control-cdn/images/atc_logo.png
2020-06-30T02:21:24.2947264Z Error                              Apache-2.0! misc/traffic-control-cdn/images/header_bkgd.png
2020-06-30T02:21:24.2947826Z Error                              Apache-2.0! misc/traffic-control-cdn/images/main_bkgd.png
2020-06-30T02:21:24.2948213Z Error                              Apache-2.0! misc/traffic-control-cdn/images/tc_logo.png
2020-06-30T02:21:24.2948622Z Error                              Apache-2.0! misc/traffic-control-cdn/images/traffic_monitor_logo.png
2020-06-30T02:21:24.2949024Z Error                              Apache-2.0! misc/traffic-control-cdn/images/traffic_operations_logo.png
2020-06-30T02:21:24.2949427Z Error                              Apache-2.0! misc/traffic-control-cdn/images/traffic_portal_logo.png
2020-06-30T02:21:24.2949826Z Error                              Apache-2.0! misc/traffic-control-cdn/images/traffic_router_logo.png
2020-06-30T02:21:24.2950201Z Error                              Apache-2.0! misc/traffic-control-cdn/images/traffic_stats_logo.png
2020-06-30T02:21:24.2950572Z Error                              Apache-2.0! test/traffic_ops_cfg/test.config
2020-06-30T02:21:24.2950952Z Error                              Apache-2.0! traffic_monitor/cache/astats.json
2020-06-30T02:21:24.2951366Z Error                              Apache-2.0! traffic_monitor/cache/stats_over_http.json
2020-06-30T02:21:24.2951852Z Error                              Apache-2.0! traffic_monitor/conf/traffic_monitor.cfg
2020-06-30T02:21:24.2952266Z Error                              Apache-2.0! traffic_monitor/conf/traffic_ops.cfg
2020-06-30T02:21:24.2952636Z Error                              Apache-2.0! traffic_monitor/peer/crstates.json
2020-06-30T02:21:24.2953003Z Error                              Apache-2.0! traffic_ops/app/conf/cdn.conf
2020-06-30T02:21:24.2953389Z Error                              Apache-2.0! traffic_ops/app/conf/development/database.conf
2020-06-30T02:21:24.2953774Z Error                              Apache-2.0! traffic_ops/app/conf/development/influxdb.conf
2020-06-30T02:21:24.2954139Z Error                              Apache-2.0! traffic_ops/app/conf/development/log4perl.conf
2020-06-30T02:21:24.2954528Z Error                              Apache-2.0! traffic_ops/app/conf/development/riak.conf
2020-06-30T02:21:24.2954905Z Error                              Apache-2.0! traffic_ops/app/conf/example-ldap.conf
2020-06-30T02:21:24.2955293Z Error                              Apache-2.0! traffic_ops/app/conf/integration/database.conf
2020-06-30T02:21:24.2955678Z Error                              Apache-2.0! traffic_ops/app/conf/integration/influxdb.conf
2020-06-30T02:21:24.2956060Z Error                              Apache-2.0! traffic_ops/app/conf/integration/log4perl.conf
2020-06-30T02:21:24.2956440Z Error                              Apache-2.0! traffic_ops/app/conf/integration/riak.conf
2020-06-30T02:21:24.2956815Z Error                              Apache-2.0! traffic_ops/app/conf/misc/psql_local.conf
2020-06-30T02:21:24.2957202Z Error                              Apache-2.0! traffic_ops/app/conf/production/database.conf
2020-06-30T02:21:24.2957594Z Error                              Apache-2.0! traffic_ops/app/conf/production/influxdb.conf
2020-06-30T02:21:24.2957980Z Error                              Apache-2.0! traffic_ops/app/conf/production/log4perl.conf
2020-06-30T02:21:24.2958345Z Error                              Apache-2.0! traffic_ops/app/conf/production/riak.conf
2020-06-30T02:21:24.2958724Z Error                              Apache-2.0! traffic_ops/app/conf/test/database.conf
2020-06-30T02:21:24.2959099Z Error                              Apache-2.0! traffic_ops/app/conf/test/influxdb.conf
2020-06-30T02:21:24.2959473Z Error                              Apache-2.0! traffic_ops/app/conf/test/log4perl.conf
2020-06-30T02:21:24.2959844Z Error                              Apache-2.0! traffic_ops/app/conf/test/riak.conf
2020-06-30T02:21:24.2960219Z Error                              Apache-2.0! traffic_ops/app/public/coverage-zone.json
2020-06-30T02:21:24.2960628Z Error                              Apache-2.0! traffic_ops/install/bin/convert_profile/convert622to713.json
2020-06-30T02:21:24.2961130Z Error                              Apache-2.0! traffic_ops/install/bin/input.json
2020-06-30T02:21:24.2961519Z Error                              Apache-2.0! traffic_ops/install/data/json/osversions.json
2020-06-30T02:21:24.2961917Z Error                              Apache-2.0! traffic_ops/testing/api/conf/traffic-ops-test.conf
2020-06-30T02:21:24.2962284Z Error                              Apache-2.0! traffic_ops/testing/api/v1/tc-fixtures.json
2020-06-30T02:21:24.2962668Z Error                              Apache-2.0! traffic_ops/testing/api/v2/tc-fixtures.json
2020-06-30T02:21:24.2963048Z Error                              Apache-2.0! traffic_ops/testing/api/v3/tc-fixtures.json
2020-06-30T02:21:24.2963416Z Error                              Apache-2.0! traffic_ops/vendor/vendor.json
2020-06-30T02:21:24.2963770Z Error                              Apache-2.0! traffic_portal/.bowerrc
2020-06-30T02:21:24.2964156Z Error                              Apache-2.0! traffic_portal/app/src/assets/images/favicon.png
2020-06-30T02:21:24.2964545Z Error                              Apache-2.0! traffic_portal/app/src/package-lock.json
2020-06-30T02:21:24.2964920Z Error                              Apache-2.0! traffic_portal/app/src/package.json
2020-06-30T02:21:24.2965402Z Error                              Apache-2.0! traffic_portal/app/src/traffic_portal_properties.json
2020-06-30T02:21:24.2965832Z Error                              Apache-2.0! traffic_portal/app/src/traffic_portal_release.json
2020-06-30T02:21:24.2966177Z Error                              Apache-2.0! traffic_portal/bower.json
2020-06-30T02:21:24.2966543Z Error                              Apache-2.0! traffic_portal/package-lock.json
2020-06-30T02:21:24.2966902Z Error                              Apache-2.0! traffic_portal/package.json
2020-06-30T02:21:24.2967296Z Error                              Apache-2.0! traffic_router/core/src/test/resources/anonymous_ip.json
2020-06-30T02:21:24.2967721Z Error                              Apache-2.0! traffic_router/core/src/test/resources/anonymous_ip_no_whitelist.json
2020-06-30T02:21:24.2968165Z Error                              Apache-2.0! traffic_router/core/src/test/resources/api/1.3/cdns/name/thecdn/sslkeys-missing-1.json
2020-06-30T02:21:24.2968603Z Error                              Apache-2.0! traffic_router/core/src/test/resources/api/1.3/cdns/name/thecdn/sslkeys.json
2020-06-30T02:21:24.2969026Z Error                              Apache-2.0! traffic_router/core/src/test/resources/api/2.0/cdns/name/thecdn/sslkeys
2020-06-30T02:21:24.2969456Z Error                              Apache-2.0! traffic_router/core/src/test/resources/api/2.0/cdns/name/thecdn/sslkeys-missing-1
2020-06-30T02:21:24.2973905Z Error                              Apache-2.0! traffic_router/core/src/test/resources/api/2.0/federations/all
2020-06-30T02:21:24.2974341Z Error                              Apache-2.0! traffic_router/core/src/test/resources/api/2.0/steering
2020-06-30T02:21:24.2974761Z Error                              Apache-2.0! traffic_router/core/src/test/resources/api/2.0/steering2
2020-06-30T02:21:24.2975132Z Error                              Apache-2.0! traffic_router/core/src/test/resources/czf.json
2020-06-30T02:21:24.2975530Z Error                              Apache-2.0! traffic_router/core/src/test/resources/czmap.json
2020-06-30T02:21:24.2976338Z Error                              Apache-2.0! traffic_router/core/src/test/resources/dczmap.json
2020-06-30T02:21:24.2976749Z Error                              Apache-2.0! traffic_router/core/src/test/resources/deliveryServices.json
2020-06-30T02:21:24.2977173Z Error                              Apache-2.0! traffic_router/core/src/test/resources/deliveryServices_missingCert.json
2020-06-30T02:21:24.2977597Z Error                              Apache-2.0! traffic_router/core/src/test/resources/deliveryServices_missingDSName.json
2020-06-30T02:21:24.2978000Z Error                              Apache-2.0! traffic_router/core/src/test/resources/keystore.jks
2020-06-30T02:21:24.2978634Z Error                              Apache-2.0! traffic_router/core/src/test/resources/publish/CrConfig.json
2020-06-30T02:21:24.2979045Z Error                              Apache-2.0! traffic_router/core/src/test/resources/publish/CrConfig2.json
2020-06-30T02:21:24.2979460Z Error                              Apache-2.0! traffic_router/core/src/test/resources/publish/CrConfig3.json
2020-06-30T02:21:24.2979866Z Error                              Apache-2.0! traffic_router/core/src/test/resources/publish/CrConfig4.json
2020-06-30T02:21:24.2980308Z Error                              Apache-2.0! traffic_router/core/src/test/resources/publish/CrStates
2020-06-30T02:21:24.2980690Z Error                              Apache-2.0! traffic_router/core/src/test/resources/regional_geoblock.json
2020-06-30T02:21:24.2981109Z Error                              Apache-2.0! traffic_server/plugins/astats_over_http/astats.config.example
2020-06-30T02:21:24.2981494Z Error                              Apache-2.0! traffic_stats/traffic_stats.cfg
2020-06-30T02:21:24.6052076Z Post job cleanup.
```
</details>

The solution would be to use v0.4, except since the workflow builds the Docker image before using it and the Docker image fails to build at git ref v0.4 (see Comcast/weasel#17). So, it would have to either use the prebuilt licenseweasel image or build the docker image at a commit at which the build succeeds and weasel passes (like `comcast/weasel@v0.4~5`).

Any other options @alficles?

Reply:
It's building at master, so a fix would fix it immediately without needing to wait for a release.

Reply:
Yeah, v0.4 broke backward compatibility with the license names. The new names are SPDX so hopefully that doesn't happen again. I updated the names in ATC in the same PR as updated the version, so it wasn't super obvious.

Reply:
v1.0 when?

Reply:
So, it looks like weasel's got a build error on the latest release. But the official way to run is via the docker image for tasks like this. I think if you change the steps to match the (very newly updated) instructions in the README, it will work: https://github.com/Comcast/weasel#github-action

The build error is now fixed on the latest master, but you don't really want to run against that. Also, you don't need to build all the dependent tools each time and almost certainly don't want to be running the full-source package which is what happens if you just run the Dockerfile directly. Here's the relevant job entry:

```jobs:
  weasel:
    runs-on: ubuntu-latest
    steps:
    - name: Checkout
      uses: actions/checkout@v2
    - name: Run weasel
      uses: docker://licenseweasel/weasel:v0.4
```

Reply:
well that's sort of why I was asking "v1.0 when?". Building from an unstable release isn't much better than master. but if the v0.4 is working then I'll update it

Reply:
The GitHub action should run the same check that `./pkg weasel` runs, which uses the prebuilt `licenseweasel/weasel:v0.4` image. We don't need to test whether Weasel builds.

Reply:
the format specified doesn't work:
```
invalid reference format
##[error]Docker pull failed with exit code 1
```

Reply:
Image still not building when I change version to 0.4, also

Reply:
Image builds at master, so I'm gonna call that our solution.